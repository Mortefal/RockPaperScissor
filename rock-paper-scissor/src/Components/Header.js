import React from 'react';
import '../Styling/Header.css';

const style = {
    display: "flex",
    alignItems: "center",
    justifyContent: "center",
    backgroundColor: "#0E0B16",
    color: "white",
    minHeight: "150px",
    fontSize: "32px",
}

const Header = () => (
  <div>
    <header style={style}>Rock-Paper-Scissors-Lizard-Spock!</header>
  </div>
);
export default Header;
