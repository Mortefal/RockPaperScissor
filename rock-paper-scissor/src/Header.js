import React from 'react';
import './Header.css'
const Header = () =>{
    return(
        <div>
            <header className="header">Rock-Paper-Scissors!</header>
        </div>
    )
}
export default Header;