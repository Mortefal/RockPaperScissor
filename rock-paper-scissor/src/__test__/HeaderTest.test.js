import Header from '../Header';
import React from "react";
import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer
        .create(<Header/>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});