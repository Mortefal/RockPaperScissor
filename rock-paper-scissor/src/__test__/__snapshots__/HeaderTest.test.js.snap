// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <header
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#0E0B16",
        "color": "white",
        "display": "flex",
        "fontSize": "32px",
        "justifyContent": "center",
        "minHeight": "150px",
      }
    }
  >
    Rock-Paper-Scissors-Lizard-Spock!
  </header>
</div>
`;
